src/mumble/Settings.cpp

```diff
@@ -825,6 +811,26 @@ void Settings::load(QSettings* settings_ptr) {
 	settings_ptr->endGroup();
 }
 
+QList<Shortcut> Settings::loadShortcuts(QSettings *settings_ptr) {
+	QList<Shortcut> qlShortcuts;
+	int nshorts = settings_ptr->beginReadArray(QLatin1String("shortcuts"));
+	for (int i=0; i<nshorts; i++) {
+		settings_ptr->setArrayIndex(i);
+		Shortcut s;
+
+		s.iIndex = -2;
```

Comment:
Why -2 ?

Reply:
That’s just moved code. It was `-2` before.

Can’t quickly see what it even is…